app/objective/post.controller.js

```diff
@@ -20,17 +44,23 @@ const validationRules = () => {
       .trim()
       .escape(),
     body('needs')
+      .custom(noNeeds)
+      .withMessage(BLANK_ERROR)
+      .bail()
       .isLength({ min: 1 })
       .withMessage('Select the needs for this objective'),
     body('needs').toArray(),
```

Comment:
Combine "body('needs')" blocks; so just add the `.toArray()` to the initial body('needs')